File: client-end/src/app/models/CustomerAndApartment.ts
export class CustomerAndApartment {
    detailsRentId: number = 0
    ApartmentId: number = 0
    custId: number = 0
    dateIncoming:Date=new Date()
    dateExit:Date=new Date()
    rentId:number=0


    constructor(detailsRentId: number,rentId:number, ApartmentId: number, custId: number
        ,dateIncoming:Date, dateExit:Date) {
        this.detailsRentId = detailsRentId
        this.ApartmentId = ApartmentId
        this.custId = custId
        this.rentId = rentId
        this.dateIncoming = dateIncoming
        this.dateExit = dateExit
    }
}

File: server/models/CustomerAndApartment.py

from sqlalchemy import Column, Integer, String, Date
from sqlalchemy.ext.declarative import declarative_base
from config import Base


class CustomerAndApartment(Base):
    __tablename__ = 'customerAndApartment'
    customerAndApartmentId = Column(Integer, primary_key=True)
    rentId = Column(Integer)
    apartmentId = Column(Integer)
    custId = Column(Integer)
    dateIncoming = Column(Date)
    dateExit = Column(Date)
